modules/artifact-registry/main.tf

```diff
@@ -197,13 +197,3 @@ resource "google_artifact_registry_repository" "registry" {
   }
 
 }
-
-resource "google_artifact_registry_repository_iam_binding" "bindings" {
```

Comment:
can you add a moved block to avoid recreating existing bindings? Check out #2607

Reply:
TIL